src/main/java/ca/team2706/frc/robot/config/Config.java

```diff
@@ -287,6 +287,8 @@ public static void init() {
 
     public static final FluidConstant<Double> ROBOT_START_ANGLE = constant("robot-start-angle-deg", 90.0);
 
+    public static final FluidConstant<Double> CONTROLLER_DEADBAND = constant("controller-deadband", 0.05);
```

Comment:
I don't think this should be a fluid constant given that it isn't used in a way which would allow for its value to update if it changed.